UnitySDK/Assets/TiltBrush/Scripts/Editor/ModelImportSettings.cs

```diff
@@ -261,19 +268,27 @@ public class ModelImportSettings : AssetPostprocessor {
         semantic == BrushDescriptor.Semantic.Position) {
       var size = GetUvsetSize(desc, uvSet);
       if (size == 3) {
-        var data = new List<Vector3>();
+        var data = new List<Vector3>(mesh.vertexCount);
         mesh.GetUVs(uvSet, data);
-        for (int i = 0; i < data.Count; ++i) {
-          data[i] = UnityFromFbx(data[i]);
```

Comment:
(here and elsewhere) Inlining like this removes the context and documentation provided by the function name; perhaps add
```
  // Converts array in-place from Fbx to Unity axis conventions
  void InPlaceUnityFromFbx(Vector{3,4}[] data)
```